view.php

```diff
@@ -159,8 +159,17 @@
                                                 $maxbytescourse,
                                                 $turnitintooltwoassignment->turnitintooltwo->maxfilesize);
 $maxfilesize = ($maxfilesize <= 0) ? TURNITINTOOLTWO_MAX_FILE_UPLOAD_SIZE : $maxfilesize;
-$turnitintooltwofileuploadoptions = array('maxbytes' => $maxfilesize,
-                                            'subdirs' => false, 'maxfiles' => 1, 'accepted_types' => '*');
+
+if ($turnitintooltwoassignment->turnitintooltwo->allownonor) {
+  $acceptedtypes = ['*'];
+}
+else {
+  $acceptedtypes = ['doc', 'docx', 'wpd', 'html', 'htm', 'ps', 
```

Comment:
where did this list of types come from?

Reply:
I created that list based on the guide found here: https://guides.turnitin.com/hc/en-us/articles/23929463501965-File-requirements

There is a list of file types in PP that I could copy from, but I am unsure whether we would have different allowed file types between the two plugins

Reply:
OK I figure it makes sense that they would be the same, so I have copied the list from the other plugin for now and pushed another commit

Reply:
please get Stephanie to confirm this list

Reply:
OK will do